### DISCUSSION TITLE
A line in aff_log.log I don't understand

### DISCUSSION THREAD

**Discussion by asotial** [Monika-After-Story/MonikaModDev#10239] [general]
I installed MAS for Linux a few days earlier, and so far I think I have been doing fine enough.
But I don't understand what happened in this line in "aff_log.log"

(After taking Monika out, then placing her back into the characters folder, and loading the game, this line appears)

`[2024-04-06 17:41:07]: !BYPASS! | greeting_returned_home | 5.0 -> 0.0 | 78.1057338474 -> 78.1057338474`

`[2024-04-06 17:41:07]`  is the date and time it happened.
`!BYPASS!` would bypass the daily affection cap.
`greeting_returned_home` is the greeting when Monika is back? (Correct me if I am wrong)
`5.0 -> 0.0` seems to be the affection gain.
`78.1057338474` -> 78.1057338474 is the change from old to new affection.

My questions are:

1: Why did `5.0 -> 0.0` happen?
2: Did I get the meaning of the line correctly?

I apologize if my English is bad, and if I have done anything wrong!

**Comment by Justformas** [comment]
Was my original answer deleted by the devs/mods? Because I didn't delete it. If so, why? I'm pretty sure it was the correct answer.

**Reply by asotial** [reply]
I saw your original answer via Gmail. Seems to be true. Thanks for the answer!

**Reply by Justformas** [reply]
Well I'm still not sure what happened to my answer, I've heard that comments can mysteriously disappear on GitHub, but glad I could help.
